src/lib/components/Header.svelte

```diff
@@ -93,6 +98,7 @@
             </div>
           {/if}
           <div class="user">
+            Collectable: {collectable}
```

Comment:
I want to improve on this. Right now it clashes with the button. I'm thinking this should live in the banner maybe on all pages or just in the dashboard. Not sure

Reply:
I undid that change actually. We should introduce a proper solution for that in a separate PR imo, together with the logic of calculating the current balanace

Reply:
Woops, I did but didn't commit. Doing now